DEPENDENCY_MANAGEMENT.md

```
@@ -0,0 +1,560 @@
+
+## Error Message
+```
+<paste error here>
+```
+
+## Expected Behavior
+What should happen
+
+## Actual Behavior
+What actually happened
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)